.github/workflows/github-actions-bazel-build.yml

```diff
@@ -0,0 +1,51 @@
+name: bazel-build
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  Build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Create Cache Timestamp
+        id: cache_timestamp
+        uses: nanzm/get-time-action@v2.0
```

Comment:
What is the benefit of using this action instead of just running the date command?

Reply:
This is just the code-snippet I have written many years ago and refine for bazel caches and used in many github CI's since, so I know it works. There are probably other ways of doing that, feel free to experiment.

Reply:
Ok, it should be fine to merge for now, but I would like to reduce the number of dependencies on external actions. It will make porting to Jenkins easier and reduce attack vectors.